vercel/turbo
# build: Update `swc_core` to `v0.96.9`

### Description

Update swc crates

### Testing Instructions


See [next.js counterpart](https://github.com/vercel/next.js/pull/67378)



 - Closes https://github.com/vercel/next.js/issues/64890
 - Closes https://github.com/vercel/next.js/issues/63104
